src/build/env.py

```diff
@@ -23,8 +23,13 @@
 else:
     from importlib import metadata
 
+
 try:
-    import virtualenv
+    virtualenv_version = packaging.version.Version(metadata.version('virtualenv'))
+    if virtualenv_version < packaging.version.Version('20.0.35'):
+        virtualenv = None
+    else:
+        import virtualenv
```

Comment:
Maybe we should make this instead of a cached function call, easier to test. Either add the pragma no coverage or additional tests. Will need a changelog too.